sidecar.sh

```diff
@@ -42,36 +41,63 @@ populate_notebook() {
 
 function populate() {
     log "querying manifest service at $GEN3_ENDPOINT/manifests/"
-    MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
-    while [ -z "$MANIFESTS" ]; do
-        log "Unable to get manifests from '$GEN3_ENDPOINT/manifests/'"
-        log $MANIFESTS
+    MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
+    log "querying manifest service at $GEN3_ENDPOINT/metadata/"
+    METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/metadata/")
+
+    while [ -z "$MANIFEST" ] && [ -z "$METADATA_FILE" ]; do
+        if [ -z "$MANIFESTE" ]; then
+            log "Unable to get manifests from '$GEN3_ENDPOINT/manifests/'"
+            log $MANIFEST
+        fi
+        if [ -z "$METADATA_FILE" ]; then
+            log "Unable to get metadata from '$GEN3_ENDPOINT/manifests/metadata/'"
+            log $METADATA
+        fi
         log "sleeping for 15 seconds before trying again.."
         sleep 15
-        MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
-    done
-
-
-    #  Loop over each exported manifest
-    echo $MANIFESTS | jq -c '.manifests[]' | while read i; do
-        FILENAME=$(echo "${i}" | jq -r .filename)
-        FOLDERNAME=$(echo "${FILENAME%.*}")
-        FOLDER="/data/${GEN3_ENDPOINT}/exported-${FOLDERNAME}"
-
-        if [ ! -d "$FOLDER" ]; then
-            log "mkdir -p $FOLDER"
-            mkdir -p $FOLDER
-
-            # make sure folder can be written to by notebook
-            chown -R 1000:100 $FOLDER
-            MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/file/$FILENAME")
-            echo "${MANIFEST}" > $FOLDER/manifest.json
-
-            log "Creating notebook for $FILENAME"
-            cp ./template_manifest.json $FOLDER/data.ipynb
-            populate_notebook "$MANIFEST" "$FOLDER"
-        fi
+        MANIFEST=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
+        METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/metadata/")
```

Comment:
```suggestion
        METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/metadata")
```
The correct path of the metadata endpoint as defined in manifestservice should be `/metadata` not `/metadata/` https://github.com/uc-cdis/manifestservice/blob/master/manifestservice/manifests/__init__.py#L208
The later path will return 404
Need to go over this sidecar script and make sure you are hitting `/metadata` but not `/metadata/`